src/Storyblok/Response.php

```diff
@@ -32,7 +32,7 @@ public function setBodyFromStreamInterface(StreamInterface $body): self
         $data = (string) $body;
         $jsonResponseData = (array) json_decode($data, true);
         // return response data as json if possible, raw if not
-        $this->httpResponseBody = $data && empty($jsonResponseData) ? $data : $jsonResponseData;
+        $this->httpResponseBody = $data && [] === $jsonResponseData ? $data : $jsonResponseData;
```

Comment:
$this->httpResponseBody = $data && $jsonResponseData === [] ? $data : $jsonResponseData;